binrz/man/rizin.1

```diff
@@ -1,132 +1,122 @@
 .Op Fl b Ar bits
+.Op Fl i Ar file
+.Op Fl s Ar addr
 .Op Fl B Ar baddr
+.Op Fl m Ar maddr
 .Op Fl c Ar cmd
 .Op Fl e Ar k=v
-.Op Fl i Ar file
-.Op Fl I Ar prefile
-.Op Fl k Ar kernel
-.Op Fl m Ar addr
-.Op Fl p Ar project
-.Op Fl P Ar patch
-.Op Fl r Ar rz-run
-.Op Fl R Ar rz-run-directives
-.Op Fl s Ar addr
-.Op Fl 0AdDwntLquvVxX
-.Ar -|--|=|file
+.Ar file|pid|-|--|=
 .Sh DESCRIPTION
-rizin is a commandline hexadecimal editor.
-.Pp
-This manpage is not updated yet. Feel free to contribute.
+       Rizin is a command-line reverse engineering framework designed to analyze and
+       disassemble binary files. It provides various tools and features for analyzing
+       and understanding the structure, behavior, and logic of executable files.
 .Pp
 The options are:
 .Bl -tag -width Fl
 .It Fl Fl
-Open rizin on an empty file
-.It Fl
-Equivalent of 'rizin malloc://512'
+run rizin without opening any file.
+.It =
+Equivalent of 'rizin malloc://512'.
+.It -
+read file from stdin.
+.It -=
+perform R=! command to run all commands remotely
 .It Fl 0
-Print \\x00 after initialization and after every command executed
+Print \\x00 after initialization and after every command executed.
 .It Fl 2
-Close stderr before starting RzCore
-.It Fl a Ar arch
-force asm.arch (x86, ppc, arm, mips, bf, java, ...)
+close stderr file descriptor (silent warning messages).
+.It Fl a Ar [arch]
+Set the assembly architecture.(x86, ppc, arm, mips, bf, java, ...).
 .It Fl A
-run 'aaa' command before prompt or patch to analyze all referenced code. Use -AA to run aaaa
-.It Fl b Ar bits
-force asm.bits (16, 32, 64)
-.It Fl B Ar baddr
-Specify the base address to be used when loading a new binary. See 'e?bin.baddr'
-.It Fl c Ar cmd
-Execute the given command before giving prompt
+run 'aaa' command before prompt or patch to analyze all referenced code.
+.It Fl b Ar [bits]
+Set the assembly bitness. (16, 32, 64).
+.It Fl B Ar [baddr]
+Set the base address for Position-Independent Executables (PIE).
+.It Fl c Ar 'cmd..'
+Execute the specified Rizin command.
+.It -C
+Specifies a file in the format "host:port" to run Rizin remotely through HTTP.
 .It Fl d
-Start in debugger mode
-.It Fl D Ar dbg.backend
-Enable debug mode. Set cfg.debug=true
+Debug the executable 'file' or running process 'pid'.
+.It Fl D Ar [backend]
+Enable debug mode, setting the configuration variable 'cfg.debug=true'.
 .It Fl e Ar k=v
-Set configuration eval variable key=value. For example \-e scr.color=false
+Evaluate the specified configuration variable.
 .It Fl f
-Blocksize = file size
-.It Fl i Ar file
-Run script file. After the file is loaded
-.It Fl I Ar file
-Run script file. Before the file is loaded
-.It Fl k Ar kernel
-Select kernel (asm.os) for syscall resolution
-.It Fl l Ar plugfile
-Load given plugin file
+Set block size to be equal to file size.
+.It -F [binplugin]
+Forces the use of a specific rbin plugin.
+.It Fl h, hh
+Shows help messages, with -hh showing detailed help.
+.It Fl H ([var])
```

Comment:
Probably worth noting that just `-H` without any argument prints useful information about the installation - location of various paths - config file, plugins, etc:
```
rizin -H
RZ_VERSION=0.7.0
RZ_PREFIX=/Users/user/.local
RZ_EXTRA_PREFIX=
RZ_MAGICPATH=/Users/user/.local/share/rizin/magic
RZ_INCDIR=/Users/user/.local/include/librz
RZ_LIBDIR=/Users/user/.local/lib
RZ_SIGDB=/Users/user/.local/share/rizin/sigdb
RZ_EXTRA_SIGDB=
RZ_LIBEXT=dylib
RZ_RCONFIGHOME=/Users/user/.config/rizin
RZ_RDATAHOME=/Users/user/.local/share/rizin
RZ_RCACHEHOME=/Users/user/.cache/rizin
RZ_LIBR_PLUGINS=/Users/user/.local/lib/rizin/plugins
RZ_EXTRA_PLUGINS=
RZ_USER_PLUGINS=/Users/user/.local/lib/rizin/plugins
RZ_IS_PORTABLE=0
```